structure_tests/types/v2/structure.go

```diff
@@ -34,11 +34,15 @@ type StructureTest struct {
 	LicenseTests       []LicenseTest
 }
 
-func (st *StructureTest) NewDriver() drivers.Driver {
-	return st.DriverImpl(st.Image)
+func (st *StructureTest) NewDriver() (drivers.Driver, error) {
+	driver, err := st.DriverImpl(st.Image)
```

Comment:
This can be collapsed into return st.Driverimpl

Reply:
ah yeah was trying something different there before but forgot to change it back